roles/ceph-common/tasks/main.yml

```diff
@@ -144,6 +144,10 @@
     (rbd_client_directory_mode is not defined or
      not rbd_client_directory_mode)
 
+- set_fact:
+    ubuntu_use_systemd: "{{ ansible_distribution_version | version_compare('16.04', '>=') }}"
```

Comment:
This could actually be a bit lower, since Ubuntu 15.04 was the first to ship systemd as the init system